app/players/elves/elf.py

```diff
@@ -0,0 +1,14 @@
+from abc import ABC
+
+from app.players.player import Player
+
+
+class Elf(Player, ABC):
```

Comment:
u have already inherited Player class from ABC
u don't need to inherit it one more time